docs/.vuepress/components/ListCard.vue

```diff
@@ -10,16 +10,16 @@
         <article :class="{ 'has-meta': item.meta }">
           <!-- h3 is overridden by the level prop. Default is 3 (h3) -->
           <h3 class="l7 mb-0 mt-0 tc-text500 art-title" :is="itemTitleTag">
-            {{ item.title }}
+            <span v-html="inlineMd(item.title)" />
```

Comment:
Remind me why this `inlineMd ` is needed? This is in order to render the twemojis?

Reply:
Yeah, inlineMd would render the emojis *and* render any inline markdown styles (italics, bold, etc)